build/compile.bat

```diff
@@ -142,11 +81,6 @@ set _platform_target=iPhone
 call :compile
 if %ERRORLEVEL% NEQ 0 if "%__ContinueOnError%" == "false" goto exit
 
-set Project=Stride.UWP.sln
-set _platform_target=UWP
-call :compile
-if %ERRORLEVEL% NEQ 0 if "%__ContinueOnError%" == "false" goto exit
-
```

Comment:
This and the changes above are to remove old/non-existent solutions from the batch file and defaults to just build stride.sln, it was the default behavior before this pr anyway since the batch exited after failing to find the stride.directx solution.